Libplanet.Explorer/Store/Models/BlockModel.cs

```diff
@@ -0,0 +1,29 @@
+namespace MySqlStore.Models
```

Comment:
Is this namespace correct?

To me, `Libplanet.Explorer.Store.Models.BlockModel` seems correct. Why doesn't it use external project(directory), `MySqlStore`?